lib/adapters/default.js

```diff
@@ -1,12 +1,11 @@
 const url = require('url');
 const uuidv4 = require('uuid/v4');
 
-const transformersPath = '../transformers';
-const mapOutRes = require(`${transformersPath}/mapOutRes`);
-const mapInReq = require(`${transformersPath}/mapInReq`);
-const mapOutReq = require(`${transformersPath}/mapOutReq`);
-const mapInRes = require(`${transformersPath}/mapInRes`);
-const mapError = require(`${transformersPath}/mapError`);
+const mapOutRes = require('../transformers/mapOutRes');
```

Comment:
maybe make an index file and require them in one go
`const { mapOutRes, mapInReq etc } = require('../transformers')`

Reply:
Done